manifests/application.pp

```diff
@@ -14,6 +14,7 @@
   String $candlepin_oauth_secret = $katello::candlepin_oauth_secret,
   Stdlib::Httpsurl $pulp_url = $katello::pulp_url,
   Stdlib::Httpsurl $crane_url = $katello::crane_url,
+  Stdlib::HTTPUrl $pulp_registry_url = $katello::pulp_registry_url,
```

Comment:
We have crane_url just above it. Isn't that the same thing? Or is this what the end user connects to?

Reply:
pulp3 acts as its own registry, so this is a separate URL

Reply:
any reason to not just rename crane_url to pulp_registry_url and just use a single setting?

Reply:
The idea being that the installer would configure the url based on pulp2 or pulp3 being available for docker. 
(Keep in mind you're getting a bit ahead of the game here, we haven't really been adding pulp3 support to the installer yet)

Reply:
@jlsherrill In the current implementation, determining whether the instance supports pulp3 and container content within pulp3 is done in Katello so both URLs are required afaict. Is there another way to go about this?

Reply:
Or that. From the code it looks like at most one will be used. This is exactly what was suggested in https://github.com/theforeman/puppet-katello/pull/243 and would have avoided this whole rename now.

That said, I'd still like to see this automatically detected rather than in a config file.